index/server/Dockerfile

```diff
@@ -36,8 +36,9 @@ RUN chmod g+rwx /var/run/nginx.pid && \
 USER www-data
 
 # Set env vars for the locations of the devfile stacks and index.json
-ENV DEVFILE_STACKS /stacks
-ENV DEVFILE_INDEX /index.json
+ENV DEVFILE_STACKS /registry/stacks
+ENV DEVFILE_SAMPLES /registry/stacks
```

Comment:
Should this be `/registry/samples`?

Reply:
Good catch, that was a bad rebase on my end 🤦